add-details-page

nik00oo opened the pull request.

nik00oo committed:
porduct details back

nik00oo committed:
back done

nik00oo committed:
back done

nik00oo committed:
back done

nik00oo committed:
Merge branch 'master' into add-details-page

# Conflicts:
#	index.js

nik00oo committed:
Merge branch 'master' into add-details-page

# Conflicts:
#	index.js

nik00oo committed:
product details back

nnatroo commented on index.js:
```diff
@@ -79,41 +79,70 @@ app.get('/products/smartphones', async (req, res) => {
+});
+
+
+app.get('/products/item/id/:id', async (req, res) => {
```
```suggestion
app.get('/products/itemId/:itemId', async (req, res) => {
```

nnatroo commented on index.js:
```diff
@@ -79,41 +79,70 @@ app.get('/products/smartphones', async (req, res) => {
+
+
+app.get('/products/item/id/:id', async (req, res) => {
+  const id = parseInt(req.params.id, 10);
```
?

nnatroo reviewed (changes requested):
Delete product images public/images folder and content

nik00oo committed:
Merge remote-tracking branch 'origin/add-details-page' into add-details-page

# Conflicts:
#	index.js

nik00oo committed:
back done

nnatroo committed:
Remove unused category SVGs and product images

Deleted category SVGs and product images that are no longer required in the project to reduce clutter and optimize storage.
